File: Baekjoon/구현/10870.py
# https://www.acmicpc.net/problem/10870

n = int(input())
dp = [0] * (n+1)
if n >= 1:
    dp[1] = 1
    if n >= 2:
        for i in range(2, n+1):
            dp[i] = dp[i-2] + dp[i-1]
        
print(dp[n])

File: Baekjoon/구현/10870.cpp
// https://www.acmicpc.net/problem/10870

#include <iostream>
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int n;
    cin >> n;
    int dp[21] = {0, 1};
    for (int i = 2; i < 21; i++) {
        dp[i] = dp[i - 1] + dp[i - 2];
    }
    cout << dp[n] << '\n';

    return 0;
}